CHANGELOG.md

```diff
@@ -1,8 +1,14 @@
+# 4.2.0
```

Comment:
lets add desc and add into doc info about finding cluster id

Reply:
Done